examples/eks.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: zarf-eks-test
```

Comment:
Suggest finding a way to add randomness to the name, else only one will be able to be active at a time in our AWS account.

Not critical for this PR so we don't creep the scope but should get added to the backlog.

Reply:
yeah that's a great point, I'm not sure what the best way to handle that is, I like providing a sample for users to just run though.  I wonder if name collisions occur per-vpc or per-account?

Reply:
I’m pretty sure it is per region

Reply:
resolved by using dynamic names now